Content.Client/UserInterface/AlertsUI.cs

```diff
@@ -9,10 +13,14 @@ namespace Content.Client.UserInterface
     /// </summary>
     public sealed class AlertsUI : Control
     {
+        public const float ChatSeparation = 38f;
         public GridContainer Grid { get; }
 
+        private readonly IChatManager _chatManager;
```

Comment:
There's no need to keep this stored considering it's just doing an event subscribe / unsubscribe, it could be resolved directly in EnteredTree / ExitedTree instead.